sdk/eventhub/azure-eventhub-checkpointstoretable/azure/eventhub/extensions/checkpointstoretable/_tablestoragecs.py

```diff
@@ -349,8 +347,6 @@ def claim_ownership(self, ownership_list, **kwargs):
                 - `partition_id` (str): The partition ID which the checkpoint is created for.
                 - `owner_id` (str): A UUID representing the owner attempting to claim this partition.
                 - `last_modified_time` (float): The last time this ownership was claimed.
-                - `etag` (str): The Etag value for the last time this ownership was modified. Optional depending
-                  on storage implementation.
```

Comment:
etag is not a thing in table?

Reply:
it is a thing in Table, but not strictly required if `table.update_entity` is called with the match condition `Unconditionally`. I do think we need it here, as per my discussion with Chris, since I'll be switching the match condition back to `IfNotModified`